admin/RELEASING.md

```diff
@@ -14,21 +26,23 @@ that needs to be remembered due to the Android component's distribution.
         * We typically leave these branches around for future maintenance releases.
     4. Run `ci/package_android_release.sh` in a UNIX compatible shell
     5. (Optional) `cargo publish -p rustls-platform-verifier-android --dry-run --alow-dirty`
-        <!---
-        TODO: Consider instead making tag-specific commits that check-in the artifacts. For now, the 
-        seamless AAR reproducibility makes this a non-issue.
-        -->
-        * `--allow-dirty` is required because we don't check-in the generated Maven local repository at this time.
+        * `--allow-dirty` is required because we don't check-in the generated Maven local repository.
     6. (Optional) Inspect extracted archive to ensure the local Maven repository artifacts are present
         1. Un-tar the `rustls-platform-verifier-android-*.crate` file inside of `target/package`.
         2. Verify `maven/rustls/rustls-platform-verifier` contains a single `*.RELEASE` directory and that contains a `.aar` file.
         3. (Optional) If the releaser has an external Gradle project that uses the configuration from the README, paste the path to the
            unzipped package's `Cargo.toml` as a replacement for the `manifestPath` variable. Run a Gradle Sync and observe everything works.
-    7. Publish the Android artifacts' new version: `cargo publish -p rustls-platform-verifier-android --alow-dirty`
+    7. **Ensure that all version changes are commited to the correct branch before proceeding**. All version increases should be checked-in prior
```

Comment:
Nit: I'd probably spell "checked-in" as just "checked in" (not a native speaker, etc)?